### PR TITLE
[WIP] Upgrade version (Java, SpringBoot, Rollbar...)

### PR DESCRIPTION
**Pull request by davidmiric**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+        <relativePath/>
+    </parent>
 
-	<groupId>ch.olmero</groupId>
-	<artifactId>rollbar-spring-boot-starter-project</artifactId>
-	<version>2.0.5.BUILD-SNAPSHOT</version>
-	<packaging>pom</packaging>
+    <groupId>ch.olmero</groupId>
+    <artifactId>rollbar-spring-boot-starter-project</artifactId>
+    <version>2.0.6.BUILD-SNAPSHOT</version>
```
Comment (dalibegovic) [inline pom.xml:15]: @StefanRankovic when we discussed this, did you said we should increment the middle one? `2.1.0`?

Comment (StefanRankovic) [reply pom.xml:15]: I believe that makes the most sense. It is a breaking change (java version), but the API and usage is largely the same.

**Comment by dalibegovic** [comment]
Since this is a public spring boot starter, we should make sure it works before we publish it. I would suggest:
1. build jar from this branch locally
2. add it as a local dependency to one of the services
3. enable rollbar in tests (set rollbar.enabled to true in application-test.properties)
4. log something from a test and see if it's sent to rollbar

File: pom.xml [hunk 2]
```diff
@@ -21,109 +21,122 @@
         </license>
     </licenses>
 
-	<properties>
-		<java.version>1.8</java.version>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<lombok.version>1.16.20</lombok.version>
-	</properties>
+    <properties>
+        <java.version>14</java.version>
```
[the remark this answers is not in the document]
Comment (davidmiric) [reply pom.xml:25]: Hi Julien. Thanks for the comment. Will do so.

**Pull request merged**